frontend/src/components/dashboard/ActivityRow.vue

```diff
@@ -9,34 +9,28 @@
       </TextAlignBaseline>
       <br />
       <CategoryChip
-        v-if="!scheduleEntry._meta.loading"
+        v-if="
```

Comment:
Maybe move these new complex v-if conditions to computeds with clear names? E.g. categoryReady or similar? Or invert it and call it categoryLoading or categorySkeleton?

Reply:
Simplified them, now that it is only an object